kubeProviders/eks/values.tmpl.yaml

```diff
@@ -5,7 +5,9 @@ jenkins-x-platform:
     # lets enable ECR docker builds
     DockerConfig: |-
       {
-        "credsStore": "ecr-login"
+        "credHelpers": {
+          "{{ .Requirements.cluster.registry }}": "ecr-login"
```

Comment:
Can you use the `credsStore` as default value when the `.Requirements.cluster.registry` is not set:

```
{{ .Requirements.cluster.registry | default "credsStore" }}
```

Reply:
Would that really work? Shouldn't it be something like the following?
```
{{- if .Requirements.cluster.registry }}
        "credHelpers": {
          "{{ .Requirements.cluster.registry }}": "ecr-login"
        }
{{- else }}
        "credsStore": "ecr-login"
{{- end  }}
```

Reply:
Yeah, This looks right to me :).

Reply:
Now done and squashed